{{cookiecutter.project_name}}/monitoring/README.md

```diff
@@ -0,0 +1,63 @@
+# Setup Model Monitoring
+
+This directory contains setup scripts intended to generate notebook that can be uploaded to databricks workspace and execute
+to set up model monitoring.
+
+## Prerequisites
+
+### Unity Catalog
+The inference table to be monitored must exist in Unity Catalog.
+If Unity Catalog is enabled for your workspace but the table exists in metastore, please run notebook code 
+or considering create workflow to copy data to Unit Catalog.
+
+For example, the following code 
```

Comment:
Expand this blurb? E.g., "the following code shows how to import your data to a Unity Catalog managed table"